tensorflow/tensorflow-engine/src/main/java/ai/djl/tensorflow/engine/TfNDArray.java

```diff
@@ -529,6 +530,66 @@ public NDArray erfinv() {
         }
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public NDArray norm() {
+        // We have to flatten first to be able to simulate "numpy.linalg.norm" whenever axis isn't
+        // specified
+        TfNDArray flattenTensor = (TfNDArray) flatten();
+        try (Tensor<?> tensor =
+                tf.linalg.euclideanNorm(flattenTensor.getOperand(), tf.constant(0)).asTensor()) {
+            return new TfNDArray(manager, tensor);
+        }
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public NDArray norm(int[] axes) {
+        try (Tensor<?> tensor =
+                tf.linalg.euclideanNorm(getOperand(), tf.constant(axes)).asTensor()) {
+            return new TfNDArray(manager, tensor);
+        }
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public NDArray norm(boolean keepDims) {
```

Comment:
```suggestion
    public NDArray norm(boolean keepDims) {
    if (dataType != DataType.FLOAT64) {
            throw new UnsupportedOperationException("float64 is not supported");
    }
    TfNDArray flattenTensor = (TfNDArray) flatten();
    try (Tensor<?> tensor =
                tf.linalg
                        .euclideanNorm(
                                flattenTensor.getOperand(),
                                tf.constant(0),
                                EuclideanNorm.keepDims(keepDims))
                        .asTensor()) {
            float number = tensor.rawData().asFloats().getFloat(0);
            long[] shapes = LongStream.generate(() -> 1).limit(shape.dimension()).toArray();
            // close the temp NDArray
            flattenTensor.close();
            return manager.create(new double[] {number}, new Shape(shapes));
    }
```

Reply:
Is there a specific reason you used `new double[]` on line 572 instead of `new float[]`? `number` is a float

Reply:
Also, thanks for this suggestion! It is much more cleaner.

Reply:
I updated the code, it should be float

Reply:
Alright, thanks again